### PR TITLE
Patch/assay mapping

### PR DESCRIPTION
**Pull request by anngvu**
- Since mapping wasn't done until snapshot of v76, this updated mapping doc accordingly
- Target model should be correctly referenced as v7.1.0, not v6.4
- Back-filled some missing assays

### CODE REVIEW

**Comment by anngvu** [comment]
@allaway I will release this as v7.1.0

**Review by allaway — CHANGES_REQUESTED** [review]
One suggestion about real-time vs qPCR, otherwise looks good!

File: NF.csv [hunk 1]
```diff
@@ -120,6 +120,11 @@ novelty response behavior assay,"A behavioral assay devised to measure novelty r
...
+in vitro tumorigenesis,"An in vitro assay devised to measure tumor formation",,,FALSE,,assay,,,,In_Vitro_Tumorigenesis,Class,Assay,Assay,Tumorigenesis_Assay,,,
+Matrigel-based tumorigenesis assay,"An assay in which tumor cells are loaded into a Matrigel matrix and measured for tumor formation characteristics",,,FALSE,,assay,,,,Matrigel-based_Tumorigenesis_Assay,Class,Assay,Assay,Tumorigenesis_Assay,,,See https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3230200/
+calcium retention capacity assay,"The CRC assay assesses Ca2+-related mitochondrial functions",,,FALSE,,assay,,,,Calcium_Retention_Capacity_Assay,Class,Assay,Assay,Cell_Assay,,,See https://pubmed.ncbi.nlm.nih.gov/29781984/
+quantitative PCR,"Quantitative PCR (Q-PCR) is used to measure the quantity of a PCR product (commonly in real-time). It quantitatively measures starting amounts of DNA, cDNA, or RNA.",,,FALSE,http://purl.obolibrary.org/obo/MI_1195,assay,,,,Quantitative_PCR,Class,Assay,Assay,RNA_Assay,,,
```
[the remark this answers is not in the document]
Comment (anngvu) [reply NF.csv:127]: Done.

**Review by allaway — APPROVED** [review]
LGTM thank you!

**Pull request merged**